### PR TITLE
swiftformat with base [MOB-2156]

### PR DESCRIPTION
**Pull request by roninopf**

### CODE REVIEW

File: sample-apps/inbox-customization/inbox-customization/Mock Data/DataManager.swift [hunk 1]
```diff
@@ -77,7 +77,8 @@ extension IterableAPI {
                                   launchOptions: [UIApplication.LaunchOptionsKey: Any]? = nil,
                                   config: IterableConfig = IterableConfig(),
                                   networkSession: NetworkSessionProtocol,
-                                  inAppFetcher: InAppFetcherProtocol) {
+                                  inAppFetcher: InAppFetcherProtocol)
+    {
```
Comment (tapashmajumder) [inline sample-apps/inbox-customization/inbox-customization/Mock Data/DataManager.swift:81]: Not sure about this. Why are we changing this?

File: swift-sdk/Internal/APNSTypeChecker.swift [hunk 2]
```diff
@@ -51,7 +51,8 @@ struct APNSTypeChecker: APNSTypeCheckerProtocol {
         } else {
             if
                 let entitlements = mobileProvision["Entitlements"] as? [AnyHashable: Any],
-                let apsEnv = entitlements["aps-environment"] as? String {
+                let apsEnv = entitlements["aps-environment"] as? String
+            {
```
Comment (tapashmajumder) [inline swift-sdk/Internal/APNSTypeChecker.swift:55]: Is this change recommended by swiftformat?

File: swift-sdk/Internal/ApiClient.swift [hunk 3]
```diff
@@ -18,7 +18,8 @@ class ApiClient {
          authProvider: AuthProvider,
          endPoint: String,
          networkSession: NetworkSessionProtocol,
-         deviceMetadata: DeviceMetadata) {
+         deviceMetadata: DeviceMetadata)
+    {
```
Comment (tapashmajumder) [inline swift-sdk/Internal/ApiClient.swift:22]: Not sure.

File: swift-sdk/Internal/ApiClient.swift [hunk 4]
```diff
@@ -91,7 +92,8 @@ extension ApiClient: ApiClientProtocol {
                   sdkVersion: String?,
                   deviceAttributes: [String: String],
                   pushServicePlatform: String,
-                  notificationsEnabled: Bool) -> Future<SendRequestValue, SendRequestError> {
+                  notificationsEnabled: Bool) -> Future<SendRequestValue, SendRequestError>
+    {
```
Comment (tapashmajumder) [inline swift-sdk/Internal/ApiClient.swift:96]: Not sure

**Comment by roninopf** [comment]
Closing to do formatting another time

**Pull request closed without merging**